estate/models/estate_property.py

```diff
@@ -0,0 +1,42 @@
+from dateutil.relativedelta import relativedelta
+from odoo import models , fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "This is property tabel."
+
+    name = fields.Char("Property Name", required=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date(string="Available From", copy=False, default=fields.Date.today()+relativedelta(months=3))
+    expected_price = fields.Float("Expected Price", required=True)
+    selling_price = fields.Float("Selling Price", readonly=True, copy=False)
+    bedrooms = fields.Integer("Bedrooms", default=2)
+    living_area = fields.Integer("Living Area")
+    facades = fields.Integer("Facades")
+    garage = fields.Boolean("Garage")
+    garden = fields.Boolean("Garden")
+    garden_area = fields.Integer("Garden Area")
+    garden_orientation = fields.Selection( 
+        string="Garden Orientation",
+        selection=[
+            ('north', 'North'), 
+            ('south', 'South'),
+            ('east', 'East'),
+            ('west', 'West')
+        ]
+    )
+    active = fields.Boolean(string="Active",default=False)
```

Comment:
```suggestion
    active = fields.Boolean(string="Active",default=False)
```
Is it necessary to define this field?

Reply:
No, removed it